<internal_error>
<sub_system>ERR</sub_system>
<callstack>
	  0x5d225b8: ccl_err + 0xa5b8 (_ZN15ERR_STACKWALKER15get_stack_traceEPPKviiPv + 0x90)
	  0x5d2576b: ccl_err + 0xd76b (_Z14err_terminatorv + 0x8a)
	  0x5ddd07e: dinkum_alt + 0x1707e (_ZSt9terminatev + 0x1e)
	  0x77b7aab: stdc++.so.6 + 0xb0aab (__cxa_rethrow + 0x5b)
	  0x63e7fbd: QtCore.so.4 + 0x17cfbd (_ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 0x36d)
	  0x63ecd0f: QtCore.so.4 + 0x181d0f (_ZN16QCoreApplication4execEv + 0xaf)
	  0x6bc8507: QtGui.so.4 + 0x154507 (_ZN12QApplication4execEv + 0x27)
	  0x804e45f: quartus + 0x645f (__gxx_personality_v0 + 0x367)
	  0x69ff908: ccl_msg + 0x23908 (_Z15msg_main_threadPv + 0x18)
	  0x5d9cf9e: ccl_thr + 0x5f9e (thr_final_wrapper + 0xe)
	  0x6a00625: ccl_msg + 0x24625 (_Z18msg_thread_wrapperPFPvS_ES_ + 0x6c)
	  0x805de1d: quartus + 0x15e1d (_Z18mem_thread_wrapperPFPvS_ES_ + 0xdd)
	  0x5d27fd8: ccl_err + 0xffd8 (_Z18err_thread_wrapperPFPvS_ES_ + 0x2a)
	  0x5d9d3fa: ccl_thr + 0x63fa (thr_thread_wrapper + 0x2f)
	  0x6a12e9a: ccl_msg + 0x36e9a (_Z12msg_exe_mainiPPKcPFiiS1_E + 0xb7)
	  0x804e531: quartus + 0x6531 (__gxx_personality_v0 + 0x439)
	  0x7d20d36: c.so.6 + 0x16d36 (__libc_start_main + 0xe6)
	  0x804e311: quartus + 0x6311 (__gxx_personality_v0 + 0x219)
</callstack>
<error>*** Fatal Error: Unhandled Exception</error>
<date>Mon Oct 27 13:59:14 2014
</date>
<version>Version 13.0.1 Build 232 06/12/2013 Service Pack 1 SJ Full Version</version>
</internal_error>